assets/src/edit-story/components/library/panes/media/mediaElement.js

```diff
@@ -113,13 +113,15 @@ const UploadingIndicator = styled.div`
  * @param {Object} param.resource Resource object
  * @param {number} param.width Width that element is inserted into editor.
  * @param {number} param.height Height that element is inserted into editor.
+ * @param {boolean} param.isMedia3p If the element is from a 3P Media provider.
  * @return {null|*} Element or null if does not map to video/image.
  */
 const MediaElement = ({
   resource,
   width: requestedWidth,
   height: requestedHeight,
   onInsert,
+  isMedia3p,
```

Comment:
This can be for later (or here), we will also need to know the provider - 'local', 'unsplash', 'coverr' etc. So this could be renamed to 'provider' and then drop down menus etc could be added conditionally if provider == 'local'.

Reply:
Added a `ProviderType` enum